src/main/ui/app/component/AddOn.jsx

```diff
@@ -1,30 +1,42 @@
 import {Component} from "react";
 import {Link} from "react-router";
+import { ListGroupItem , Badge, Accordion , Panel , Media , Button } from 'react-bootstrap';
 
 export default class AddOn extends Component {
 
     render() {
         const addon = this.props.addon;
         const icon = addon.type === 'OWA' ? 'fa fa-2x fa-globe' : 'fa fa-2x fa-puzzle-piece';
-        const link = this.props.version ? `/show/${addon.uid}?highlightVersion=${this.props.version}` : `/show/${addon.uid}`;
+        const link = this.props.version ? `#/show/${addon.uid}?highlightVersion=${this.props.version}` : `#/show/${addon.uid}`;
```

Comment:
The initial path was index/#/(Not sure why) and hence without this change none of the other pages would load,hence I had to add this